tests/unit/test_mock.py

```diff
@@ -29,33 +34,47 @@
     mock_SpikeEventSeries,
 )
 
-from pynwb.testing import TestCase
 import pytest
 
+from pynwb.testing.mock.utils import name_generator
+
+
+@pytest.mark.parametrize(
+    "mock_function", [
+        mock_ImagingPlane,
+        mock_TwoPhotonSeries,
+        mock_RoiResponseSeries,
+        mock_PlaneSegmentation,
+        mock_OpticalChannel,
+        mock_Fluorescence,
+        mock_DfOverF,
+        mock_ImageSegmentation,
+        mock_OptogeneticStimulusSite,
+        mock_OptogeneticSeries,
+        mock_Device,
+        mock_Position,
+        mock_PupilTracking,
+        mock_CompassDirection,
+        mock_SpatialSeries,
+        mock_ElectrodeGroup,
+        mock_ElectrodeTable,
+        mock_ElectricalSeries,
+        mock_SpikeEventSeries,
+        mock_Subject,
+        mock_NWBFile,
+        mock_TimeSeries,
+    ],
+)
+def test_mock(mock_function):
+    mock_function()
+
+
+def test_name_generator():
+
+    from pynwb.testing.mock.utils import name_generator_registry
```

Comment:
Does this import statement need to be in the function?

Reply:
yes, I think it does

Reply:
I get that the `clear()` needs to be here, but I am pretty sure that in this case, the import statement can be outside the function. I tested this locally and in a simplified MWE and it works fine. 

`from pynwb.testing.mock.utils import name_generator, name_generator_registry`